caasp-csi-livenessprobe-image/caasp-csi-livenessprobe-image.kiwi

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v4.5/livenessprobe:%%PKG_VERSION%%-rev<VERSION> caasp/v4.5/livenessprobe:%%PKG_VERSION%%-rev<VERSION>-build<RELEASE> caasp/v4.5/livenessprobe:beta -->
+
+<image schemaversion="6.9" name="caasp-csi-livenessprobe-image" xmlns:suse_label_helper="com.suse.label_helper">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>livenessprobe running on a SLES15 SP2 container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.2">
+      <containerconfig
+        name="caasp/v4.5/livenessprobe"
+        tag="%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <entrypoint execute="/usr/bin/livenessprobe"/>
```

Comment:
I'm not sure will you change the binary name to `csi-livenessprobe`, leave it to you :smile:

Reply:
I've left the binary as it is. Should be ok.